src/types/entities/guild.rs

```diff
@@ -59,7 +59,7 @@ pub struct Guild {
     pub emojis: Vec<Shared<Emoji>>,
     pub explicit_content_filter: Option<ExplicitContentFilterLevel>,
     //#[cfg_attr(feature = "sqlx", sqlx(try_from = "String"))]
-    pub features: Option<GuildFeaturesList>,
```

Comment:
This change causes a deserialization error, which is why the authenticate test fails: 

```Failed to parse gateway event READY (missing field `features` at line 1 column 3753)```

Reply:
Having the option there is dumb though, since it's `Vec<GuildFeatures>`. Maybe we can fix this with     `#[serde(default)] ` (since we have `Default` derived)